apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/components/CardStylingSettings.tsx

```diff
@@ -194,107 +79,236 @@ export const CardStylingSettings = ({
         <hr className="py-1 text-slate-600" />
 
         <div className="flex flex-col gap-6 p-6 pt-2">
-          <div className="flex max-w-xs flex-col gap-4">
-            <div className="flex flex-col">
-              <h3 className="text-sm font-semibold text-slate-700">Roundness</h3>
-              <p className="text-xs text-slate-500">Change the border radius of the card and the inputs.</p>
-            </div>
-            <div className="flex flex-col justify-center rounded-lg border bg-slate-50 p-6">
-              <Slider value={[roundness]} max={22} onValueChange={(value) => setRoundness(value[0])} />
-            </div>
+          <div className="flex flex-col justify-center">
+            <FormField
+              control={form.control}
+              name="roundness"
+              render={({ field }) => (
+                <FormItem>
+                  <div>
+                    <FormLabel className="text-sm font-semibold text-slate-700">Roundness</FormLabel>
```

Comment:
The Roundness toggle is not working correctly in my tests and not updating the UI properly.

<img width="1260" alt="Screenshot 2024-05-28 at 21 40 39" src="https://github.com/formbricks/formbricks/assets/675065/339ceb88-480e-4c13-b0bd-227cc6409c18">